### ISSUE TITLE
How to specify path of GoogleService-Info.plist  in Workflow Step?

### ISSUE THREAD

**Issue by sathish211** [andrewmarmion/bitrise-step-firebase-dsym-upload#2]
Hi,

can you please let me know how i can specify GoogleService-Info.plist path in Workflow's Step. I checked in my code, i mentioned the path in Step of Workflow and it is not working. Below is the screenshot for reference. Please let me know
<img width="1392" alt="Screen Shot 2019-11-01 at 4 47 35 PM" src="https://user-images.githubusercontent.com/50426877/68600850-dd0dcb00-0468-11ea-839f-eebffaf854af.png">

Thanks,
Sathish

**Comment by andrewmarmion** [comment]
That’s seems strange. Looking at your folder location it looks like it should work. There is nothing in the code that would cause it to not find the file. 

The file path is usually `./<APP NAME>/GoogleService-Info.plist`

Are you sure that the GoogleService-Info.plist is located there and that you have spelt everything correctly? Is the file checked into your repo? Or if not is it copied to the correct location as part of your pipeline? Are there any spaces in the directory path or file names?

**Comment by sathish211** [comment]
@andrewmarmion . Thanks for your response. my app name is ⁨loxabeauty-stylistui and i have GoogleService-InfoUSStaging.plist file under following folder.

⁨loxabeauty-stylistui ▸ ⁨LoxaBeauty⁩ ▸ ⁨GoogleService⁩ ▸ ⁨GoogleService-InfoUSStaging.plist

I checked the folder name and file name are correct. I had check-in the file in my repo. i don't have space but i have hypen in folder name and file name.

i have tried ./⁨loxabeauty-stylistui/⁨LoxaBeauty⁩/⁨GoogleService⁩/⁨GoogleService-InfoUSStaging.plist and ./⁨LoxaBeauty⁩/⁨GoogleService⁩/⁨GoogleService-InfoUSStaging.plist path but i am getting below error for both. i have created extra folder GoogleService⁩ and change the file name as well. I am not sure whether it is issue. Please let me know.

![Screen Shot 2019-11-11 at 11 52 09 AM](https://user-images.githubusercontent.com/50426877/68609432-e6a02e80-047a-11ea-8859-b5772611c1fa.png)

**Comment by andrewmarmion** [comment]
There is a check in the code that makes sure that the file exists, otherwise the upload command will error out if there is no file present. The only reason that would return false is if it doesn't find your file. 

Hyphens in the file name shouldn't pose an issue as the default location for fabric contains a hyphen.

It's difficult for me to debug what is going on as it is your project and you should know where your files are located. 

To check if your location is correct you could always add a  `script` step that checks the location of the file (not ideal but it is a quick and dirty way to make sure you have the correct location). Put that step after your `git clone` if it fails then you know you have the wrong location, saves you having to wait  for a full build.  Something like this should do the trick

```
if [ ! -e ./⁨loxabeauty-stylistui/⁨LoxaBeauty⁩/⁨GoogleService⁩/⁨GoogleService-InfoUSStaging.plist ]; then
  echo "Could not find your plist"
  exit 1
else 
  echo "Found your plist"
fi
```
This is basically the same check that is in the step.

**Comment by sathish211** [comment]
@andrewmarmion thanks for the script. I figured out what is problem

![Screen Shot 2019-11-11 at 4 29 43 PM](https://user-images.githubusercontent.com/50426877/68626308-bd929480-04a0-11ea-9d87-acc0e8719e56.png)

Looks like some special characters got introduced in the path which i cannot notice it but when i copy same path to my texteditor i able to see it. I found special character in your script output. I removed the special character and it is working fine. Thanks for your help

Thanks,
Sathish
